tests/manager/manager-ctor-ssl-002.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+PHPC-1239: Passing SSL driverOptions overrides SSL options from URI
```

Comment:
If I understand correctly, this test is only significant when we're connecting over SSL, since MO will add SSL parameters in the URI string. I realize my previous PR review suggested creating this test, but I just want to sanity check as my mind has wandered since I last reviewed :)

Reply:
It's relevant only for SSL, but that doesn't mean we can't spend the 0.0005 seconds to run it on the other topologies...